RealmTasks Android/app/src/main/java/io/realm/realmtasks/SignInActivity.java

```diff
@@ -89,6 +92,9 @@ public void onClick(View view) {
 
     private void loginComplete(User user) {
         UserManager.setActiveUser(user);
+
+        createInitialDataIfNeeded();
```

Comment:
No, since `loginComplete(User)` is called even when the account has been registered.

In `RegisterActivity#registrationComplete(User)`, it calles `UserManager.setActiveUser(user);` and then restart `SignInActivity`.

Then `SignInActivity#onCreate()` calls `loginComplete(user);`